### PR TITLE
Add OTP per action

### PR DESCRIPTION
**Pull request by franciscocardoso**

### CODE REVIEW

File: _entities.md [hunk 1]
```diff
@@ -320,6 +320,24 @@ type | The type of endpoint. Possible values are 'email', 'card' and 'external'.
       "numberFormat": {
```
Comment (fixe) [inline _entities.md]: Lets remove `hasOtpEnabled` from the documentation since we will not support it in the future.

File: _entities.md [hunk 2]
```diff
@@ -416,6 +434,14 @@ type | The type of endpoint. Possible values are 'email', 'card' and 'external'.
 
 The `user` object contains all information we have on record relating to the currently logged in user.
 
+<aside class="warning">
```
Comment (fixe) [inline _entities.md]: We will handle this particular communication through a different channel, you can remove this section. /cc @ruimarinho

File: _entities.md [hunk 3]
```diff
@@ -424,6 +450,13 @@ Property  | Description
 --------- | -----------
 memberAt  | The date when the user has become a verified member.
 
+### User Settings
+
+* **otp.login.enabled** - This will prompt the user to input an OTP token on login.
```
Comment (fixe) [inline _entities.md]: ... OTP token when creating an OAuth token?

File: _entities.md [hunk 4]
```diff
@@ -424,6 +450,13 @@ Property  | Description
...
+* **otp.transactions.send.enabled** - This will prompt the user to input an OTP token when transacting to other users.
```
Comment (fixe) [inline _entities.md]: ... when creating a transaction to another user?

File: _entities.md [hunk 5]
```diff
@@ -424,6 +450,13 @@ Property  | Description
...
+* **otp.transactions.transfer.enabled** - This will prompt the user to input an OTP token when transacting between the user's own cards.
+* **otp.transactions.withdraw.crypto.enabled** - This will prompt the user to input an OTP token when withdrawing to the bitcoin network.
```
Comment (fixe) [inline _entities.md]: Bitcoin at the moment -- I don't mind keeping as is.

File: _entities.md (hunk 1 above)
Comment (franciscocardoso) [inline _entities.md]: :+1:

**Pull request merged**